timeout/timeout.c

```diff
@@ -398,10 +399,12 @@ void settimeout(struct TClock *duration) {
 }
 
 int validate_signal(char *str) {
-	int i;
+	register int i;
```

Comment:
No.

Reply:
I hate the Antiregister.

Reply:
O.k., changed opinion on this: I think removing the registers would be pretty nice, but I think we could make another pull-request for refactoring other parts. Nawk also needs fixes for building on some versions of GCC that will complain about mismatching types.
What do you think?
cc.: @deablofk (you said you wanted to finish your refactoring work)